### PR TITLE
issue: Create MCQA for this Object #25

### PR DESCRIPTION
**Pull request by JP-GECT**

### CODE REVIEW

File: code/035/script.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    };
+  }
+  
+  const user1 = new User('kirandash');
+  const user2 = new User('bgwebagency');
+  
+  user1.updateUsername('openai');
+  user2.updateUsername('gpt-3');
+  
+  console.log(user1.username + user2.username);
```
Comment (kirandash) [inline code/035/script.js:15]: Please update this code to match with code from README.md file

```diff
- function User(username) {
-     this.username = username;
-   
-     this.updateUsername = (newName) => {
-       this.username = newName;
-     };
-   }
-   
-   const user1 = new User('kirandash');
-   const user2 = new User('bgwebagency');
-   
-   user1.updateUsername('openai');
-   user2.updateUsername('gpt-3');
-   
-   console.log(user1.username + user2.username);
  
+ function User(username) {
+   this.username = username
+   this.updateUsername = (newName) => {
+     this.username = newName
+   }
+ }
+
+ const user1 = new User('kirandash')
+ const user2 = new User('bgwebagency')
+
+ user1.updateUsername('-website')
+ user2.updateUsername('-app')
+
+ console.log(user1.username + user2.username)
```

File: README.md [hunk 2]
```diff
@@ -1274,6 +1274,50 @@ Finally, the method `sayHello` is called on the `person` object using dot notati
...
+<details><summary><b>Answer</b></summary>
+<p>
+
+#### Answer: B
+
+The code defines a constructor function User that creates user objects with a username property and an updateUsername method. Two user objects, user1 and user2, are created with initial usernames 'kirandash' and 'bgwebagency' respectively.
+
+The updateUsername method is called on both user1 and user2 objects to update their usernames. user1's username is updated to '-website', and user2's username is updated to '-app'.
+
+Finally, the code logs the concatenation of user1.username and user2.username, which results in the output 'kirandash-app'.
```
Comment (kirandash) [inline README.md:1314]: Please add proper formatting for code

```diff
- The code defines a constructor function User that creates user objects with a username property and an updateUsername method. Two user objects, user1 and user2, are created with initial usernames 'kirandash' and 'bgwebagency' respectively.
- 
- The updateUsername method is called on both user1 and user2 objects to update their usernames. user1's username is updated to '-website', and user2's username is updated to '-app'.
- 
- Finally, the code logs the concatenation of user1.username and user2.username, which results in the output 'kirandash-app'.

+ The code defines a constructor function `User` that creates `user` objects with a `username` property and an `updateUsername` method. Two `user` objects, `user1` and `user2`, are created with initial usernames 'kirandash' and 'bgwebagency' respectively.
+ 
+ The `updateUsername` method is called on both `user1` and `user2` objects to update their usernames. `user1`'s username is updated to '-website', and `user2`'s username is updated to '-app'.
+ 
+ Finally, the code logs the concatenation of `user1.username` and `user2.username`, which results in the output 'kirandash-app'.

**Review by kirandash — CHANGES_REQUESTED** [review]
@JP-GECT Thanks for the PR! 

It's awesome. I have requested some minor changes. pls have a look

File: code/035/script.js (hunk 1 above)
Comment (JP-GECT) [reply code/035/script.js:15]: Thank You!
I will make the changes.

**Pull request closed without merging**